monorepo/monorepo-ws/apps/articleapp/src/Schema/interfaces.ts

```diff
@@ -1,6 +1,13 @@
+import { Maybe } from "graphql/jsutils/Maybe"
 
 export interface Article {
     id: number,
     title: string,
     authorId: number
-}
\ No newline at end of file
+}
+
+export interface GqlUser {
```

Comment:
❓ tohle je vygenerovane nebo jsi to napsal sam?

Reply:
To jsem psal já. Tento interface používám k vytvoření testovacího resolveru pro uživatele z druhého subgrafu. Za jeho pomoci mohu provolat přez usera jeho articles.

Reply:
👍 jasně pro teď ok, jinak pojďme tam dodat to generování interfaces automaticky by nějakej task, je to v tom druhém projektu graphql-codegen